content/developer/addons/addon-info.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Addon Information
+tags:
+- Developers
+- Addons
+- Configuration
+- about.php
+- addon.json
+category: addons
+menu:
+  developer:
+    weight: 10
+    parent: addons
+version:
```

Comment:
The global versioning is not working because the readme is out of date. Use: "versioning" instead of "version". I'll update the readme. Issue: https://github.com/vanilla/docs/issues/147